packages/api-client/package.json

```diff
@@ -9,7 +9,7 @@
   "dependencies": {
     "@koh/common": "*",
     "axios": "^0.21.1",
-    "class-transformer": "^0.3.1",
+    "class-transformer": "0.3.1",
```

Comment:
`class-transformer`'s latest version has some breaking changes, so I decided to freeze it for now